rules/hmap.bzl

```
@@ -51,13 +55,21 @@ def _make_headermap_impl(ctx):
 
     :return: provider with the info for this rule
     """
-    hdrs_lists = [ctx.files.hdrs]
+    hdrs_lists = [[(s, d) for (s, d) in zip(ctx.files.hdrs, ctx.attr.hdr_dests)]]
```

Can we just pass `zip` directly?

```suggestion
    hdrs_lists = [zip(ctx.files.hdrs, ctx.attr.hdr_dests)]
```

---

Yeah, no idea. I think it used to be a slightly different structure 😅